scijava/scijava-ops-engine/src/main/java/org/scijava/ops/engine/impl/DefaultOpEnvironment.java

```diff
@@ -337,7 +337,7 @@ private Type[] toTypes(Nil<?>... nils) {
 	/**
 	 * Creates an Op instance from an {@link OpInfo} with the provided
 	 * {@link MatchingConditions} as the guidelines for {@link OpInfo} selection.
-	 * This Op instance is put into the {@code opCache}, and is retrievable via
+	 * This Op instance is put into the {@link #opCache}, and is retrievable via
```

Comment:
By default when you generate javadoc, private members are not included in the generated output. So then this sort of link, which goes to a private field, will not actually be functional. I am actually not sure how this ends up rendering in the default case, but we should check. If it looks bad or confusing, we can just remove the `@link` in favor writing "op cache" in normal language. But if it looks OK, I favor leaving it so that in IDEs the linkage is still there.